src/main/java/org/bigredbands/mb/views/ProjectView.java

```diff
@@ -1294,8 +1293,7 @@ public void clearSelectedRanks() {
      */
     public void updateCommandList(ArrayList<CommandPair> commands) {
         commandListView.setCommands(commands);
-        saved = false;
-        updateProjectTitle();
+        mainView.updateProjectTitle();
```

Comment:
While this technically isn't necessary (you can still call `updateProjectTitle` directly in `ProjectView`), this makes sense to set up eventually moving the ownership of the project title out of `ProjectView`.

Reply:
Ah, this was technically a copy-and-paste error - I originally fully moved updateProjectTitle into MainView before realizing that wouldn't work given the project's current layout.  I'd actually lean towards removing the `mainView.` here; whatever future refactoring is done is going to be major enough that setting up something like this probably won't matter.